frontend/src/router.js

```diff
@@ -17,7 +18,10 @@ const router = new VueRouter({
     { path: "/screening", component: Screening },
     { path: "/matching", component: Matching },
     { path: "/student/:id", component: StudentProfile },
-    { path: "/new", component: NewStudent },
+    { path: "/new-student", component: NewStudent },
+    { path: "/students", component: Students },
```

Comment:
Technically, you cannot include this until the PR with the `Students.vue` is merged. So you should chase people to review and get that merged! 😃